File: android/versioned-abis/expoview-abi42_0_0/src/main/java/abi42_0_0/expo/modules/haptics/HapticsPackage.java
package abi42_0_0.expo.modules.haptics;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi42_0_0.org.unimodules.core.BasePackage;
import abi42_0_0.org.unimodules.core.ExportedModule;
import abi42_0_0.org.unimodules.core.ViewManager;

public class HapticsPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new HapticsModule(context));
  }
}

File: android/versioned-abis/expoview-abi46_0_0/src/main/java/abi46_0_0/expo/modules/haptics/HapticsPackage.kt
package abi46_0_0.expo.modules.haptics

import android.content.Context
import abi46_0_0.expo.modules.core.BasePackage

class HapticsPackage : BasePackage() {
  override fun createExportedModules(context: Context) =
    listOf(HapticsModule(context))
}
